r-package/grf/R/average_partial_effect.R

```diff
@@ -91,13 +111,13 @@ average_partial_effect = function(forest, calibrate.weights = TRUE, subset = NUL
   
   # Estimate variance using the calibration
   if (length(subset.clusters) == 0) {
-    cape.se = sqrt(mean((debiasing.weights * plugin.residual)^2) / length(subset.W.orig))
+    cape.se = sqrt(mean((debiasing.weights * plugin.residual)^2) / (length(subset.W.orig) - 1))
```

Comment:
I'm curious why we've added this `- 1` here?

Reply:
It's by analogy to <a href="https://en.wikipedia.org/wiki/Bessel%27s_correction">Bessel's correction</a> for variance estimation. Doesn't really make a difference in large samples, but still the right thing to do.